frame/composable-traits/src/staking/mod.rs

```diff
@@ -43,12 +146,12 @@ pub trait ProtocolStaking {
 	type AccountId;
 	type AssetId;
 	type Balance;
-	type PoolId;
+	type RewardPoolId;
```

Comment:
Not sure, I am ok with PoolId in context of Staking. Parent namespace defines what pool id is. Not sure why to mix and produce repetion. Runtim(Protocol(Pallet(Pool(Position))))

Reply:
This is to prevent confusions between Pablo pools and staking pools, it's for human readability

Reply:
not sure, for me confusing double naming. I do not think it is good naming.